clinic.v1.yaml

```diff
@@ -72,11 +72,23 @@ paths:
             type: string
           in: query
           name: sortOrder
+        - schema:
+            type: string
+          in: query
+          name: clinicianId
+        - schema:
+            type: string
+          in: query
+          name: patientId
       security:
         - ClinicAuth:
             - TIDEPOOL_ADMIN
+            - CLINIC_ADMIN
```

Comment:
The prescription service will not make requests on behalf of clinicians or patients to the clinic endpoint - it will make requests using a service token, so having service only authentication should be ok.

Reply:
Good to know - will take this out - this was going to be a bit of a problem